src/SystemPaths.cc

```diff
@@ -117,6 +146,26 @@ std::string SystemPaths::LogPath() const
 void SystemPaths::SetPluginPathEnv(const std::string &_env)
 {
   this->dataPtr->pluginPathEnv = _env;
+
+  // TODO(CH3): Deprecated. Remove on tock.
+  std::string result;
+  if (!this->dataPtr->pluginPathEnv.empty())
+  {
+    if (env(this->dataPtr->pluginPathEnv, result))
+    {
+      // TODO(CH3): Deprecated. Remove on tock.
+      std::string ign_prefix = "IGN_";
+
+      // Emit warning if env starts with IGN_
+      if (env_.compare(0, ign_prefix.length(), ign_prefix) == 0)
```

Comment:
Compilation is failing here:

```
/home/jenkins/workspace/ignition_common-ci-pr_any-ubuntu_auto-amd64/ign-common/src/SystemPaths.cc:160:11: error: ‘env_’ was not declared in this scope
  160 |       if (env_.compare(0, ign_prefix.length(), ign_prefix) == 0)
      |           ^~~~
```

Reply:
`env_` -> `_env` :man_facepalming: 
https://github.com/gazebosim/gz-common/pull/365/commits/7222e6b81141e3be809cd89463d81432278cc7df

Reply:
I didn't fix the other one, oops..

https://github.com/gazebosim/gz-common/pull/365/commits/26f7f5b5115f36eaaf221d05b97e2fa32d3d9143